sql/src/test/java/ddlog/JooqProviderTestBase.java

```diff
@@ -687,4 +687,14 @@ public void testIndexSelect() {
         assertTrue(readFromInput.contains(test3));
     }
 
+    @Test
+    public void testBigInt() {
```

Comment:
no tests that validate the output DDlog?

Reply:
I don't think that's necessary. This test fails if you run without the patch.